src/OpenTelemetry.Instrumentation.GrpcNetClient/GrpcClientInstrumentationOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="GrpcClientInstrumentationOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Instrumentation.GrpcNetClient
+{
+    /// <summary>
+    /// Options for GrpcClient instrumentation.
+    /// </summary>
+    public class GrpcClientInstrumentationOptions
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether instrumentation is suppressed (disabled).
+        /// </summary>
+        public bool SuppressInstrumentation { get; set; }
```

Comment:
Must be named something like "SupressChildHttpInstrumentation". And description should explain the impact. @alanwest can probably suggest alternate wordings :)

Reply:
@alanwest , Do you think `SupressChildHttpInstrumentation` is a best name to use?

Reply:
Good question. `SuppressChildHttpInstrumentation` is ok.

Another thought is to go with something more generic like `SuppressDownstreamInstrumentation`. Thought being that we may want to apply this pattern to other instrumentation and this name starts a commonly understood convention for suppressing downstream instrumentation.

Reply:
WCF is an example of an RPC framework that supports multiple underlying protocols that may also be instrumented. Would it make sense to have a separate config setting for each of the protocols?